annotate.el

```diff
@@ -458,71 +457,71 @@ position (so that it is unchanged after this function is called)."
     (point)))
 
 (defun annotate-annotated-text-empty-p (annotation)
-  "Does this `ANNOTATION' contains annotated text?"
+  "Does this ANNOTATION contains annotated text?"
   (= (overlay-start annotation)
      (overlay-end   annotation)))
 
 (defun annotate-annotation-set-face (annotation face)
-  "Set property face to `FACE' for `ANNOTATION'."
+  "Set property face to FACE for ANNOTATION."
   (overlay-put annotation 'face face))
 
 (defun annotate-annotation-face (annotation)
-  "Get property face from `ANNOTATION'."
+  "Get property face from ANNOTATION."
   (overlay-get annotation 'face))
 
 (defun annotate-annotation-set-annotation-face (annotation face)
-  "Set property annotation-face to `FACE' for `ANNOTATION'."
+  "Set property annotation-face to FACE for ANNOTATION."
   (overlay-put annotation 'annotation-face face))
 
 (defun annotate-annotation-property-annotation-face (annotation)
-  "Get property annotation-face from `ANNOTATION'."
+  "Get property annotation-face from ANNOTATION."
   (overlay-get annotation 'annotation-face))
 
 (defun annotate-annotation-set-annotation-text (annotation annotation-text)
-  "Set the annotation's content for `ANNOTATION` to `ANNOTATION-TEXT`."
+  "Set the annotation's content for ANNOTATION to ANNOTATION-TEXT."
   (overlay-put annotation 'annotation annotation-text))
 
 (defun annotate-annotation-get-annotation-text (annotation)
-  "Get the annotation's content for `ANNOTATION`."
+  "Get the annotation's content for ANNOTATION."
   (overlay-get annotation 'annotation))
 
 (defun annotate-annotation-set-position (annotation position)
-  "Set the annotation's position policy for `ANNOTATION'
-to the value bound to `POSITION'."
+  "Set the annotation's position policy for ANNOTATION
+to the value bound to POSITION."
   (overlay-put annotation 'annotate-position position))
 
 (defun annotate-annotation-get-position (annotation)
   "Get the annotation's position policy for `ANNOTATION'."
   (overlay-get annotation 'annotate-position))
 
 (defun annotate-overlay-maybe-set-position (overlay position)
-  "Set the annotation's position policy for `ANNOTATION' to the value bound
-to `POSITION',but only if the value of the property \\='position is not null."
+  "Set the annotation's position policy for OVERLAY to the value bound
+to POSITION,but only if the value of the property `position' is not null."
   (when position
     (annotate-annotation-set-position overlay position)))
 
 (defun annotate-chain-last-ring (chain)
-  "Get the last ring of `CHAIN'."
+  "Get the last ring of CHAIN."
   (car (last chain)))
 
 (defun annotate--remap-chain-pos (annotations)
-  "Remap `ANNOTATIONS' as an annotation \"chain\".
+  "Remap ANNOTATIONS as an annotation \"chain\".
 
 An annotation is a collection of one or more overlays that
-contains the property `ANNOTATE-PROP-CHAIN-POSITION'.
+contains the property `annotate-prop-chain-position'.
```

Comment:
why aren't these upcased?

Reply:
As far as i have understood, upcased text is for function arguments only, symbols supposed to be downcased and wrapped.

I am going to recheck the code, because i fear i have made some mistakes.

Bye!
C.

Reply:
looks like i did. :sweat_smile: 

Bye!
C.